HousingRepairsOnlineApi/Gateways/IGovNotifyGateway.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using HousingRepairsOnlineApi.UseCases;
+
+namespace HousingRepairsOnlineApi.Gateways
+{
+    public interface IGovNotifyGateway
```

Comment:
Would there be more than one Gov implementation of this interface?
If not, changing the name to be generic (i.e. without "Gov") would be more intuitive to allow additional potential implementations.

```suggestion
    public interface INotifyGateway
```

(Note: this change would need to also update references of this class)